.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+indent_style = tab
```

Comment:
sounds like this was determined to be removed, based on the description and @matthewroach's previous comment on tabs here

```
.editorconfig
We may want to disregard the .editorconfig file if it's causing issues or is unneeded. This was added this temporarily to prevent indentation errors in our editors. It will most-likely be removed prior to the final PR, unless wanted or needed.
```

previous comment: https://github.com/WPMedia/arc-themes-components/pull/50#discussion_r827974319

Reply:
@JackHowa : Indeed. `.editorconfig` has been removed from this PR. Thanks!